include/experimental/__p1673_bits/blas1_linalg_add.hpp

```diff
@@ -132,7 +132,7 @@ void add_rank_2(
   using size_type = std::common_type_t<SizeType_x, SizeType_y, SizeType_z>;
   for (size_type j = 0; j < x.extent(1); ++j) {
     for (size_type i = 0; i < x.extent(0); ++i) {
-      z(i,j) = x(i,j) + y(i,j);
+      z[i,j] = x[i,j] + y[i,j];
```

Comment:
The actual implementation is a C++17 back-port, so we unfortunately have to roll with whatever operator (parentheses or brackets) the user has available.  If you really do want to change the implementation, then we might have to go with something like the following.

```suggestion
#if (MDSPAN_USE_PAREN_OPERATOR > 0)
            z(i,j) = x(i,j) + y(i,j);
#else
            z[i,j] = x[i,j] + y[i,j];
#endif
```

or at least

```suggestion
#if defined(__cpp_multidimensional_subscript)
            z[i,j] = x[i,j] + y[i,j];
#else
            z(i,j) = x(i,j) + y(i,j);
#endif
```

It might be best just to leave the implementation alone; we might want to come up with a better way to do this.